packages/core/test/rules/obsolete_statement.ts

```diff
@@ -74,6 +76,58 @@ END OF li_order.`, cnt: 1, fix: false},
 
 testRule(tests, ObsoleteStatement);
 
+async function findIssues(abap: string, version?: Version): Promise<readonly Issue[]> {
+  const config = Config.getDefault(version);
+  const reg = new Registry(config).addFile(new MemoryFile("zfoo.prog.abap", abap));
+  await reg.parseAsync();
+  const rule = new ObsoleteStatement();
+  return rule.initialize(reg).run(reg.getFirstObject()!);
+}
+
+describe("test obsolete_statements rule - versions", () => {
+  it("statements no issues", async () => {
+    const issue1 = await findIssues("FIND REGEX 'foo' IN 'bar'.", Version.v754);
```

Comment:
I don't know how to run the check using different version, other than this.
I remember there being a way to do it with testRule, but I don't know how.

Reply:
this should work okay for this scenario?

Reply:
This works fine.